examples/playground.php

```diff
@@ -3,7 +3,6 @@
     $url = @$_POST['url'] ?: @$_GET['url'];
     $sel = @$_POST['sel'] ?: @$_GET['sel'];
     $go  = @$_POST['go']  ?: @$_GET['go'];
-    $ch  = @$_POST['ch']  ?: @$_GET['ch'];
```

Comment:
`$_GET['ch']` allow to save the state in the URL and preserve it during refreshes. Maybe not a big deal for the cache expires, but I'd prefer to have consistency over all variables.

Reply:
Eg. `http://localhost:8074/playground.php?url=http://duzun.me&sel=a%20img:parent&ch=10`.